.travis.yml

```diff
@@ -0,0 +1,60 @@
+language: go
+
+go:
+  - '1.12.x'
+env:
+  global:
+    - GO111MODULE=on
+stages:
+  - name: test
+  - name: release-staging
+    if: branch = staging AND type = push
+  - name: deploy-staging
+    if: branch = staging AND type = push
+  - name: release-production
+    if: tag IS present
+  - name: deploy-production
+    if: tag IS present
+
+go_import_path: github.com/src-d/landing
+
+jobs:
+  include:
+    - name: 'Test'
+      stage: test
+      env:
+        - LANDING_URL="//sourced.tech"
+      script:
+        - make build
```

Comment:
`packages` target already depends on `build` one, so it's not needed to run it in advance, is it?
If removed, it should speed up a bit this and next stages.

Reply:
Removed them!